File: src/main/java/Classes/Person.java
package Classes;

public class Person {
    private int id;
    private String fio;
    private Long cardNumber;
    private Long hashPass;
    private String login;

    public Person(int id, String fio, Long cardNumber, Long hashPass, String login){
        this.id = id;
        this.fio = fio;
        this.cardNumber = cardNumber;
        this.hashPass = hashPass;
        this.login = login;
    }

    public int getId(){
        return this.id;
    }
    public String getFio(){
        return this.fio;
    }
    public Long getHashPass(){
        return this.hashPass;
    }
    public void setHashPass(Long hashPass){
        this.hashPass = hashPass;
    }
    public String getLogin(){
        return this.login;
    }
    public void setLogin(String login){
        this.login = login;
    }
}

File: src/main/Python/person.py
class Person:
    id: int
    fio: str
    card_number: long
    hash_pass: long
    login: str

    def __init__(self, id, fio, card_number, hash_pass, login):
        self._id = id
        self._fio = fio
        self._card_number = card_number
        self._hash_pass = hash_pass
        self._login = login

    @property
    def id(self):
        return self._id

    @property
    def fio(self):
        return self._fio

    @property
    def hash_pass(self):
        return self._hash_pass

    @hash_pass.setter
    def hash_pass(self, hash_pass):
        self._hash_pass = hash_pass

    @property
    def login(self):
        return self._login

    @login.setter
    def login(self, login):
        self._login = login
